JoinColony/colonyCDapp
# [Multi-sig] feat: implement multi sig motion creation and support for minting tokens

## Description

Implement creating a multiSig motion, currently support just for minting tokens.

## Testing

1. Install the MultiSig permissions extension.
2. Open the action creator
3. Create a mint tokens multiSig action
4. Verify that the multiSig motion was created by running this query:
```
query MyQuery {
  listColonyMultiSigs {
    items {
      id
      transactionHash
      createdAt
      action {
        type
        amount
      }
    }
  }
}
```
5. The UI should also redirect you to the transaction hash and a placeholder component should be displayed in the sidebar

## Diffs

**New stuff** ✨

* `createRootMultiSig` saga
* dummy `MultiSigSidebar` component\
* support for creating a `mintTokens` multi sig

**Changes** 🏗

* Multi-sig added to decision methods dropdown

Resolves #2150
